src/and-cli-dotnet-test.ts

```diff
@@ -28,12 +28,17 @@ CommandRunner.run(async () => {
             "-s, --skip-clean",
             "Skips the clean, build, and restore steps before running the dotnet test runner. This will speed up sequential runs if intentionally running on the same assemblies."
         )
+        .option(
+            "-w, --watch",
+            "Run dotnet test in watch mode. Will only work when run on a single project (.csproj)."
```

Comment:
Does the project need to be supplied, or will it require you to `cd` into the appropriate directory?

Reply:
Short answer, yes.  I was debating on adding an additional project argument and validation that they are both supplied. 

Currently, it will only work with something like this: 
`and-cli dotnet-test -w -- DictionaryExtensionsTest --project .\test\AndcultureCode.CSharp.Extensions.Tests\AndcultureCode.CSharp.Extensions.Tests.csproj` 

Ideally it would be something like this so we could validate that `--project` is also supplied whenever `--watch` is set:
`and-cli dotnet-test -w --project .\test\AndcultureCode.CSharp.Extensions.Tests\AndcultureCode.CSharp.Extensions.Tests.csproj --filter DictionaryExtensionsTest`

Reply:
Gotcha 👍 I'm fine with keeping it simple for now, but it might be worth it to update the description of the flag to better describe this behavior or provide an example until/if we wrap those options ourselves.